components/server/src/ide-config.ts

```diff
@@ -190,6 +191,23 @@ export class IDEConfigService {
                 }
             }
 
+            for (const [id, option] of Object.entries(newValue.ideOptions.options).filter(([_, x]) => x.latestImage)) {
+                if (option.latestImage == null || option.latestImage === "") {
```

Comment:
```suggestion
                if (option.latestImage === null || option.latestImage === "") {
```

I think it's safer, given that `undefined == null` => `true`